config/rubocop/.rubocop.yml

```diff
@@ -512,7 +512,7 @@ Style/CommandLiteral:
 Style/CommentAnnotation:
   Description: >-
                  Checks formatting of special comments
-                 (TODO, FIXME, OPTIMIZE, HACK, REVIEW).
+                 (T-ODO, F-IXME, O-PTIMIZE, H-ACK, R-EVIEW).
```

Comment:
I think this format is more confusing. Can we just delete the line? There's a url to explain this more if people are confused. Maybe we could also change the description to `Checks for presence of maintenance comments`? WDYT?